docs/ANALYTICS.md

```diff
@@ -0,0 +1,24 @@
+# SDS Analytics
+
+These samples showcase some commonly performed data visualization of SDS data.
+
+## [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md)
+
+These samples highlight basic operations of Data Views for OCS, including creation, updating, getting data from and deletion of Data Views. Please see the [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md) readme for more information on this sample.
+
+## [Data Views Jupyter](https://github.com/osisoft/sample-ocs-data_views_jupyter-python)
+
+This sample demonstrates how to utilize OCS Data Views to do some basic data analysis using Python Jupyter Notebook. Please see the [Data Views Jupyter](https://github.com/osisoft/sample-ocs-data_views_jupyter-python) readme for more information on this sample.
+
+## [Data Views R](https://github.com/osisoft/sample-ocs-data_views_r-r)
+
+This sample Demonstrates how to create a data frame in R from an OCS Data View. These actions include creating, deleting, updating and reading. Please see the [Data Views R](https://github.com/osisoft/sample-ocs-data_views_r-r) readme for more information on this sample.
```

Comment:
This phrasing isn't quite right. The sample only shows how to read from a data view. It shows how to create a data frame, and I guess the library's code show how to iteratively update a data frame, but I'm not sure that's what was meant. Either way, it's not really a CRUD sample.